src/routes/auth/v1/account/+page.svelte

```diff
@@ -56,6 +57,42 @@
 			});
 		}
 	});
+
+	const providerLinkPkce = async (provider_id: string, pkce_challenge: string) => {
+		const data = {
+			pkce_challenge,
+			redirect_uri: window.location.href,
+			client_id: 'rauthy',
+			email: userInfo?.email,
+			provider_id,
+		}
+		await fetch(`/auth/v1/providers/${provider_id}/link`, {
+			method: 'POST',
+			headers: [['csrf-token', localStorage.getItem('csrfToken')!]],
+			body: JSON.stringify(data),
+		}).then(() => {
+			getPkce(64, async (error, { challenge, verifier }) => {
+				if (!error) {
+					localStorage.setItem(PKCE_VERIFIER, verifier);
+					const nonce = getKey(24);
+					const s = 'account';
+					const redirect_uri = `${window.location.origin}/auth/v1/oidc/callback`
+						.replaceAll(':', '%3A')
+						.replaceAll('/', '%2F');
```

Comment:
Why are these replace calls necessary?

And if they are necessary, should we use `encodeUriComponent` instead?